README.md

```diff
@@ -98,7 +98,7 @@ pipenv lock
 
 ## Contributing ##
 
-We welcome contributions!  Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for
+We welcome contributions! Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for
```

Comment:
We try to avoid changing things from the skeleton unnecessarily.

```suggestion
We welcome contributions!  Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for
```